firmware/version_data.py

```diff
@@ -0,0 +1,117 @@
+import  subprocess
+import  os
+import  tempfile
+import  filecmp
+import  shutil
+
+commit          = subprocess.Popen(['git', 'log', '--format="%H"', '-n''1'], stdout=subprocess.PIPE)
```

Comment:
Please use [PEP8 formatting](https://www.python.org/dev/peps/pep-0008/) for this file.